spec/services/nufs_account_builder_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe NufsAccountBuilder, type: :service do
+  let(:builder) { described_class.new(options) }
+
+  it "is an AccountBuilder" do
+    expect(described_class).to be <= AccountBuilder
+  end
+
+  describe "#build override" do
+    let(:options) do
+      {
+        account_type: "NufsAccount",
+        facility: build_stubbed(:facility),
+        owner_user: build_stubbed(:user),
+        current_user: build_stubbed(:user),
+        params: params,
+      }
+    end
+
+    let(:params) do
+      ActionController::Parameters.new({
+        nufs_account: {
+          account_number: "1234",
+          description: "foobar",
+        }
+      })
+    end
+
+    it "sets expired_at" do
+      expect(builder.build.expires_at).not_to be_nil
```

Comment:
That change was made.  :) https://github.com/tablexi/nucore-open/commit/3b55d49db7e0b9254f2504975f870be179c21fed#diff-df5031acd0a4fcde49a8612d191d1505R31